go/sample-apps/function/build.sh

```diff
@@ -4,6 +4,7 @@ set -e
 GOARCH=${GOARCH-amd64}
 
 mkdir -p build
+export CGO_ENABLED=0 
 GOOS=linux go build -o ./build/bootstrap .
```

Comment:
```suggestion
CGO_ENABLED=0 GOOS=linux go build -o ./build/bootstrap .
```

This should probably include `GOARCH` as well, though I'm not sure the setting of `GOARCH` on line 4 is correct.

Reply:
It does pick the env var `GOARCH` even-though it is set above. Looks like `GOARCH` is set to `amd64` if it is not defined. When building the layer, we explicitly declare the value of `GOARCH` before running `./build.sh`. One of such example is [here](https://github.com/aws-observability/aws-otel-lambda/blob/5870d5cde57134f25b211bbf67f9b60fc91fccfc/.github/workflows/pr-build.yml#L80). So we should be good right?